src/clr-angular/data/tree-view/tree-node.ts

```diff
@@ -144,6 +160,7 @@ export class ClrTreeNode<T> implements OnInit, OnDestroy {
       this._model.selected.pipe(filter(() => !this.skipEmitChange)).subscribe(value => this.selectedChange.emit(value))
     );
     this.subscriptions.push(this.expandService.expandChange.subscribe(value => this.expandedChange.emit(value)));
+    this.subscriptions.push(this._model.disabled.subscribe(value => this.disabledChange.emit(value)));
```

Comment:
@gnomeontherun: Be careful with this, it has the same behavior as the selection that caused problems and that you very recently fixed. It's the async event emitter + the output that fires when you receive an input. I will let you handle it, I think your fix for selection should work fine here too.

Reply:
Indeed, once we have the flag setup per my comment above, this line should be

```
    this.subscriptions.push(
      this._model.disabled.pipe(filter(() => !this.skipEmitChange)).subscribe(value => this.disabledChange.emit(value))
    );
```